src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -7797,8 +8290,8 @@ namespace HVACVariableRefrigerantFlow {
         Real64 AirMassFlow;         // total supply air mass flow [m3/s]
         int OpMode;                 // fan operating mode, DataHVACGlobals::CycFanCycCoil or DataHVACGlobals::ContFanCycCoil
         int VRFCond;                // index to VRF condenser
-        Real64 SpecHumOut;          // specific humidity ratio at outlet node
-        Real64 SpecHumIn;           // specific humidity ratio at inlet node
+        Real64 SpecHumOut(0.0);     // specific humidity ratio at outlet node
+        Real64 SpecHumIn(0.0);      // specific humidity ratio at inlet node
```

Comment:
Uninitialized variables were popping up before wait for air loop to be read in was implemented. Probably don't need this anymore, probably doesn't hurt.